nomad-base/src/settings/chains.rs

```diff
@@ -185,6 +208,7 @@ impl ChainSetup {
                 )
                 .into())
             }
+            ChainConf::Substrate(_) => unimplemented!("TODO"),
```

Comment:
Are we leaving this intentionally for now?

Reply:
Yep